config.md

```diff
@@ -61,7 +61,7 @@ This specification uses the following terms:
 Here is an example image JSON file:
```

Comment:
[Update the title][1] to:

> # Image Configuration

[1]: https://github.com/opencontainers/image-spec/pull/255#discussion_r77895393